js/src/components/free-listings/setup-free-listings/form-content.js

```diff
@@ -47,15 +47,15 @@ const FormContent = ( {
 			<ConditionalSection show={ shouldDisplayTaxRate }>
 				<TaxRate />
 			</ConditionalSection>
-			<StepContentFooter>
+			<StepContentActions>
```

Comment:
Now that you've separated the `StepContentActions` from the `StepContentFooter`, I think it would be good to be consistent about wrapping `StepContentActions` in a `StepContentsFooter` component everywhere it's used, rather than just when it's in a Section with a `FaqPanel`. What do you think?

Reply:
Handled in [0a1f579](https://github.com/woocommerce/google-listings-and-ads/pull/2531/commits/0a1f57920157f274c568fbef952e8509ac8e2f19)

Reply:
Agree @joemcgill 👍